src/t8_geometry/t8_geometry_implementations/t8_geometry_cad.cxx

```diff
@@ -921,6 +916,219 @@ t8_geometry_cad::t8_geom_evaluate_cad_hex (t8_cmesh_t cmesh, t8_gloidx_t gtreeid
   T8_FREE (interpolated_coords);
 }
 
+void
+t8_geometry_cad::t8_geom_evaluate_cad_prism (t8_cmesh_t cmesh, t8_gloidx_t gtreeid, const double *ref_coords,
+                                             const size_t num_coords, double *out_coords) const
+{
+  T8_ASSERT (active_tree_class == T8_ECLASS_PRISM);
```

Comment:
I can only roughly check this function as I am not an CAD expert. Can maybe @sandro-elsweijer have another look at it?